lib/mutant/integration/rspec.rb

```diff
@@ -35,6 +37,7 @@ class Rspec < self
       #
       def initialize(*)
         super
+        @all_expression = expression_parser.call(ALL_EXPRESSION)
```

Comment:
It looks like in some places you use `expression_parser.(...)` and in some places you use `expression_parser.call(...)`.

I won't flag all of them, but I wanted to bring it to your attention. I'm wondering if there are other callsites for other objects where one or the other can be used too.